### PR TITLE
Update copyright date on all file under src/

### PR DESCRIPTION
**Pull request by sickpig**
Done via:

    ./contrib/devtools/copyright_header.py update src/

### CODE REVIEW

**Comment by sickpig** [comment]
Seems to be something that the account we use for janitor tasks should. So feel free to close this one and redo the changes using such account.

File: src/qt/receivefreezedialog.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2015-2016 The Bitcoin Unlimited developers
+// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
 // Distributed under the MIT software license, see the accompanying
```
Comment (ptschip) [inline src/qt/receivefreezedialog.cpp:2]: missed one here

File: src/qt/receivefreezedialog.h [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2015-2016 The Bitcoin Unlimited developers
+// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
 // Distributed under the MIT software license, see the accompanying
```
Comment (ptschip) [inline src/qt/receivefreezedialog.h:2]: missed one, wrong date

File: src/wallet/test/accounting_tests.cpp [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2012-2015 The Bitcoin Core developers
-// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
+// Copyright (c) 2015-2016 The Bitcoin Unlimited developers
 // Distributed under the MIT software license, see the accompanying
```
Comment (ptschip) [inline src/wallet/test/accounting_tests.cpp:3]: missed one.

File: src/qt/unlimiteddialog.h [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The Bitcoin Unlimited developers
+// Copyright (c) 2011-2017 The Bitcoin Unlimited developers
 // Distributed under the MIT software license, see the accompanying
```
Comment (ptschip) [inline src/qt/unlimiteddialog.h:2]: another

File: src/qt/unlimiteddialog.cpp [hunk 5]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2015 The Bitcoin Unlimited developers
+// Copyright (c) 2011-2017 The Bitcoin Unlimited developers
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
```
Comment (ptschip) [inline src/qt/unlimiteddialog.cpp:3]: another

**Comment by sickpig** [comment]
From what I have understand the script update the copyright date, e.g. if you look at the git log history of ` src/qt/unlimiteddialog.cpp ` you can verify that the last time it has been modified was: 

```git
commit 481fe403be9549aa6b2210902cdbfb8bb2b5a373
Author: BitcoinUnlimited Janitor <BitcoinUnlimitedJanitor@bitcoinunlimited.info>
Date:   Thu May 18 17:42:52 2017 -0400

    source formatting
```

hence the script substituted 2015 w/ 2017.

I'm not sure if this is the right things to do here but it seems logic to do so.

**Pull request merged**